src/verifytime_main.c

```diff
@@ -20,5 +20,5 @@
 
 int main()
 {
-	return verify_time() ? 1 : 0;
+	return verify_time() ? 0 : 1;
```

Comment:
I think it is a good idea to add comments in both files explaining why you are returning false instead of true since it is confusing, so we don't forget why is like that in the future

Reply:
ok. that's true